Update DD_DOGSTATSD_TAG_CARDINALITY behavior (#6933)
DataDog/documentation: content/en/developers/dogstatsd/unix_socket.md

Origin detection allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received via UDS are tagged by the same container tags as Autodiscovery metrics.

> To configure [tag cardinality][13] for the metrics collected using origin detection, set the environment variable `DD_DOGSTATSD_TAG_CARDINALITY` to `low`, `orchestrator`, or `high`. The default is `low`.

Reviewer: ```suggestion
To configure [tag cardinality][13] for the metrics collected using origin detection, set the environment variable `DD_DOGSTATSD_TAG_CARDINALITY` to `low` (default), `orchestrator`, or `high`.
```

> **Note:** `container_id`, `container_name`, and `pod_name` tags are not added automatically to avoid creating too many [custom metrics][11].

Reviewer: ```suggestion
**Note:** `container_id`, `container_name`, and `pod_name` tags are not added by default to avoid creating too many [custom metrics][11].
```

[10]: https://github.com/DataDog/datadog-agent/wiki/Unix-Domain-Sockets-support
[11]: /developers/metrics/custom_metrics
[12]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_definition_pidmode

> [13]: /tagging/assigning_tags/?tab=agentv6v7#environment-variables

Reviewer: ```suggestion
[13]: /tagging/assigning_tags/#environment-variables
```
